geoextent/lib/helpfunctions.py

```diff
@@ -98,32 +97,53 @@ def getDelimiter(csv_file):
     csv_file.seek(0)
     return csv.reader(csv_file.readlines(), delimiter=dialect.delimiter)
 
-def date_parser_iso8601(list):
+def get_time_format(time_list):
     '''
-    Function purpose: help-function to transform tbox from extracted time format into ISO8601
-    Input: list (tbox) \n
-    Output: array values
+    Function purpose: 'Guess' time format of a list of 'strings' by taking a representative sample
+    Input: list of strings \n
+    Output: time format in string format (e.g '%Y.%M.d')
     '''
 
-    time_1, time_2 = None, None
-    time_form = [time_format(np.array([list[0]])), time_format(np.array([list[1]]))]
-
-    if time_form[0] == time_form[1]:
-        time_1 = pd.to_datetime(list[0], format=time_form[0])
-        time_2 = pd.to_datetime(list[1], format=time_form[0])
+    date_time_format = None
+    num_sample = 30
```

Comment:
Please make this a function parameter with a default. If users use the Python API of the library, they might want to change `num_sample`. They should be able to change such a parameter as part of the main extraction function, too.

Reply:
Ok !